### PR TITLE
Feature/add single machine

### PR DESCRIPTION
**Pull request by eileenabadi**
FE:
-. create adding single machine page
-. align styling with current styling

BE:
-. create connection to DB

### CODE REVIEW

File: client/src/app/shared/forms/new-machine-form/new-machine-form.constant.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { FormControl, Validators } from '@angular/forms';
+
+export const NewMachineFormFields = {
+  applicationEnvironment: new FormControl('', [
+    Validators.required,
```
Comment (maxiemwagenaar) [inline client/src/app/shared/forms/new-machine-form/new-machine-form.constant.ts]: there is a comma after every Validators.required

File: client/src/app/main/account/account.module.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { AccountListComponent } from './list/account-list.component';
 import { AccountNewComponent } from './new/account-new.component';
 import { SharedModule } from '../../shared/modules/shared.module';
 import { AccountEditComponent } from './edit/account-edit.component';
+import {AngularFontAwesomeModule} from "angular-font-awesome";
```
Comment (maxiemwagenaar) [inline client/src/app/main/account/account.module.ts]: No space between the name of the module and the { }
The same is happening in other files down below

File: client/src/app/main/account/edit/account-edit.component.html [hunk 3]
```diff
@@ -16,13 +25,14 @@
...
-        <div class="col-10">
-          <input id="form-password" class="form-control" type="{{showPassword?'text':'password'}}"
-                 formControlName="password">
+      <div class="form-group row form-bottom-margin">
+        <div class="col-md-2 offset-md-8 ">
+          <input class="margin-left" type="checkbox" (click)="hideShow()">Show Password
+        </div>
+        <label class="col-md-3 offset-md-2 col-form-label">Password</label>
+        <div class="col-md-5">
+          <input id="form-password" class="form-control" type="{{showPassword?'text':'password'}}" formControlName="password">
```
Comment (mischaboldy) [inline client/src/app/main/account/edit/account-edit.component.html]: can you use `[type]="showPassword?'text':'password'"` ? find it slightly cleaner

File: client/src/app/main/account/edit/account-edit.component.html [hunk 4]
```diff
@@ -35,11 +45,11 @@
       </div>
     </ng-container>
     <ng-container *ngIf="editAccountForm.get('confirmPassword')">
-      <div class="form-group row">
-        <label class="col-2 col-form-label">Confirm Password</label>
-        <div class="col-10">
+      <div class="form-group row form-bottom-margin">
+        <label class="col-md-3 offset-md-2 col-form-label">Confirm Password</label>
+        <div class="col-md-5">
           <input id="form-confirm-password" class="form-control" type="{{showPassword?'text':'password'}}"
```
Comment (mischaboldy) [inline client/src/app/main/account/edit/account-edit.component.html]: same comment as above

**Pull request merged**